GirLoader/TypeReferenceResolver.cs

```diff
@@ -32,7 +32,7 @@ public bool Resolve(Model.TypeReference typeReference, Model.Repository reposito
             return false;
         }
 
-        private bool TryResolveAlias(Model.TypeReference typeReference, Model.Repository repository, [MaybeNullWhen(false)] out Model.Type type)
+        private bool TryResolveAlias(Output.TypeReference typeReference, Output.Repository repository, [MaybeNullWhen(false)] out Output.Type type)
         {
             type = null;
```

Comment:
Is `type = null` necessary? It seems a bit redundant if RecursiveResolveAlias returns null anyway.